AgentDiscoveries-Frontend/app/sass/styles.scss

```diff
@@ -33,6 +33,10 @@ body {
   margin-bottom: 0.75rem;
 }
 
+.report-textarea-noresize {

```

Comment:
Are we happy with this name?
If you're naming a css class after both the relevant component & the intent (e.g. report-textarea is the component, but no-resize is the intent), I'd be inclined to break them up line component-name--intent-details
E.g. `.report-textarea--no-resize` ?